DCS_STATUS/StatusVmon904HV.py

```diff
@@ -0,0 +1,1266 @@
+                elif len( vmonIDToUse ) > 1:
+                	query = "select TS,VALUE_NUMBER from EVENTHISTORY where ("
+                	for strIdx in range ( len( vmonIDToUse ) ):
+                		query = query + "ELEMENT_ID = "+str( vmonIDToUse[ strIdx ] )
+                		if ( len( vmonIDToUse ) > 1 and strIdx < ( len( vmonIDToUse ) - 1 ) ):
+                			query = query + " or "
+                	query = query + ") and TS > to_date ("+sta_period+",'YYYY-MM-DD HH24:MI:SS') and TS < to_date ("+end_period+",'YYYY-MM-DD HH24:MI:SS')"
+                		
+                print query 
+                cur.execute(query)                                                                                                                             
+
+		#result[0] is the present TS
+		#result[1] is the voltage
+		curvmon=cur
+		vmonRec=[]
+		vmonOnlyT = array ( 'd' )
+                vmonOnlyV = array ( 'd' )
+                vmonOnlyTDate = array ( 'd' )
+		contatoreV = 0
+		isNotEmptyVmon = False
+		for result in curvmon:
+		   #start ts: the first read Ts
+		   if contatoreV == 0: 
+	   	   	#print("dentroV", contatoreV)
+		   	startTsVmon = result[0]
+		   isNotEmptyVmon = True
+		   VmonTh1List[counter].Fill(result[1])
+		   vmonRec+=[(result[0],result[1],"Vmon")]
+		   #print("startTsVmon",startTsVmon, "present ts", result[0])
+		   currentTsVmon = result[0] #put the present TS value in currentTsVmon
+		   #print("TSVmon", currentTsVmon-startTsVmon)
+		   #micro=(currentTsVmon-startTsVmon).microseconds
+		   tot_secondsVmon = (currentTsVmon-startTsVmon).total_seconds()
+		   #print("micro=", micro)
+		   #print("totsecVmon=", tot_secondsVmon)
+		   tot_milliVmon=tot_secondsVmon*1000.
+		   #print("totmilliVmon=", tot_milliVmon)
+		   #print(vmonRec[contatoreV])
+		
+		   #lists with only V and t
+                   vmonOnlyT.append(tot_secondsVmon)
+                   vmonOnlyV.append(result[1])
+
+		   vmonOnlyTDate.append(4.) #an any float number
+                                                                                                        
+                   #print currentTsVmon
+                   year = str(currentTsVmon)[0:4]
+                   #print year
+                   month = str(currentTsVmon)[5:7]
+                   #print month
+                   day = str(currentTsVmon)[8:10]
+                   #print day
+                   hour = str(currentTsVmon)[11:13]
+                   #print hour
+                   minute = str(currentTsVmon)[14:16]
+                   #print minute
+                   second = str(currentTsVmon)[17:]
+                   #print second
+                   micro = str(currentTsVmon)[20:]
+                   #print micro
+                                                                                                       
```

Comment:
consider using a python native library for this, e.g., [this](https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior) and [this](https://docs.python.org/2/library/datetime.html#datetime.datetime.strptime)